### DISCUSSION TITLE
Device Manager 404 - not found -> pin_mapping.json does not exist

### DISCUSSION THREAD

**Discussion by der-dave** [tbnobody/OpenDTU#1944] [Q&A] [answered]
Hello guys, thank you very much for all your awkword work. I found the beginning thread on Mikrocontroller.net and cant believe you "hacked" the protocol.

For now i got a problem with my device. When opening the "Device-Manager" page I only got a 404 error - Page noch found. Via Network Traffic and in console logs I already found out that there is a problem with the pin_mapping.json.

Here the full log:
> [171305][E][vfs_api.cpp:105] open(): /littlefs/pin_mapping.json does not exist, no permits for creation

Any ideas how to solve this? Is it possible to add the file manually?

I am using a AZ-Delivery ESP32 38 pin with ESP-WROOM-32 and a NRF24L01+ plus display unit. But I am sure it has nothing to do with my hardware. Tried another power supply and double checked wiring and so on.

I am using last version [v24.4.12](https://github.com/tbnobody/OpenDTU/releases/tag/v24.4.12)

Thanks for your help.

**Comment by tbnobody** [comment, marked as answer]
Seems that you found a bug. Will fix it till the next release. (soon)

fixed in [v24.4.24](https://github.com/tbnobody/OpenDTU/releases/tag/v24.4.24)

**Comment by der-dave** [comment]
Thanks a lot for fixing the bug so fast. It is working now as it has to 👍
